Stripe/PublicHeaders/STPPaymentContext.h

```diff
@@ -148,8 +148,16 @@ NS_ASSUME_NONNULL_BEGIN
 
 /**
  *  The user's shipping address. May be nil.
- */
-@property(nonatomic, readonly, nullable)STPAddress *shippingAddress;
+ *  If you've already collected a shipping address from your user, you may
+ *  prefill it by setting this property. When your user enters a new address,
+ *  PaymentContext will save it to the current customer object. When PaymentContext
+ *  loads, if you haven't manually set a prefilled value, any shipping information
+ *  saved on the customer will be used to prefill shippingAddress.
+ *  Note: by default, your user will still be prompted to verify a prefilled 
+ *  shipping address. To change this behavior, you can set 
+ *  `verifyPrefilledShippingAddress` to NO in your `STPPaymentConfiguration`.
+ */
+@property(nonatomic, nullable)STPAddress *shippingAddress;
```

Comment:
I think it's somewhat confusing that the shipping prefill is on payment context, but all the other prefills are in the prefilled user info block. I think we should reconsider this structure. I think this might be a good time to also clear up how properties that are listed in multiple places are used (like phone/email)